session_items.py

```diff
@@ -13,9 +14,14 @@ def get_items():
     Returns:
         list: The list of saved items.
     """
-    return session.get('items', _DEFAULT_ITEMS)
-
-
+    todos = requests.get(
+        "https://api.trello.com/1/lists/5f38fc74a51a6723b3fe75ea/cards?key=58750588def43275eb1a4457a8efe87d&token=fea90c8a2c8ebf27103def869abcf71abfa536b4d119f8bf3cfd680463f7226f").json()
```

Comment:
3 values in here should not be shared - the list id, the API "key" and the API "token".

You should get those values from config and construct the URL.

Also, I expected you to make a request to `https://api.trello.com/1/boards/{board_id}/cards` rather than `...lists/{list_id}/cards` because you want to get all cards on the board, of any status. The list will represent the status, e.g. "To Do" or "Completed".

Reply:
secret values are now been read from config file